docs/stdlib/datetime.rst

```diff
@@ -256,39 +253,38 @@ EdgeDB stores and outputs timezone-aware values in UTC.
         select <cal::local_time>'15:01:22.306916';
         select <cal::local_time>'15:01:22';
 
-    Note that when casting from strings, the string should be in ISO
-    8601 time format.
+    When casting :eql:type:`cal::local_time` from strings, the string must be
+    follow the ISO8601 format.
 
-    See functions :eql:func:`cal::time_get`, :eql:func:`cal::to_local_time`,
-    and :eql:func:`to_str` for more ways of working with
-    :eql:type:`cal::local_time`.
+    For more information regarding interacting with this type, see
+    :eql:func:`datetime_get`, :eql:func:`cal::to_local_time` and
+    :eql:func:`to_str`.
 
 
 ----------
 
 
-
 .. _ref_datetime_duration:
 
 .. eql:type:: std::duration
 
-    A type representing a span of time.
+    Represents a timespan.
 
-    Valid units when converting from a string (and combinations of them):
-    ``'microseconds'``,
-    ``'milliseconds'``,
-    ``'seconds'``,
-    ``'minutes'``,
-    ``'hours'``.
+    A :eql:type:`duration` is a fixed number of seconds and microseconds and
+    isn't adjusted by timezone, length of month or anything else in datetime
+    calculations.
+
+    When converting from a string, only units of ``'microseconds'``,
+    ``'milliseconds'``, ``'seconds'``, ``'minutes'`` and ``'hours'`` are
+    valid:
 
     .. code-block:: edgeql
 
         select <duration>'45.6 seconds';
         select <duration>'15 milliseconds';
         select <duration>'48 hours 45 minutes';
```

Comment:
Can we change this to a repl block so we can also show an example of attempting to cast a duration from an unsupported unit (like months) and show the error that raises? We would also then want to add the results of these valid casts.